### DISCUSSION TITLE
How to make fullscreen in react-native-windows?

### DISCUSSION THREAD

**Discussion by nomad-pixel** [microsoft/react-native-windows#9586] [Q&A] [answered]
I have a windows application on react-native-windows and I need to make it fullscreen. How can I do this

**Comment by chrisglein** [comment, marked as answer]
Related question posed here: #9047
Documentation on how to create a fullscreen UWP app [here](https://docs.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.applicationview.tryenterfullscreenmode?view=winrt-22000)
